.vsts-ci.yml

```diff
@@ -8,59 +11,70 @@ resources:
     ref: refs/heads/master
 
 variables:
-  ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+  teamName: Roslyn-Project-System
+  ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
Either change "in" to "eq" if you think this list of reasons will never change, or change "ne" to "notIn" (below), if you think this may get updated in the future.  The point is to just be consistent.

eq + ne = for string comparison
in + notIn = for list comparison